components/script/textinput.rs

```diff
@@ -607,14 +547,14 @@ impl<T: ClipboardProvider> TextInput<T> {
     /// adjusted vertically and the process repeats with the remaining adjustment requested.
     pub fn adjust_horizontal(
         &mut self,
-        adjust: UTF8Bytes,
+        adjust_by_bytes: usize,
```

Comment:
It's weird that we still need to deal with bytes here. Shouldn't we always be dealing with code units or characters?